internal/internal_workflow_testsuite.go

```diff
@@ -404,11 +406,16 @@ func (env *testWorkflowEnvironmentImpl) executeWorkflowInternal(delayStart time.
 		panic(err)
 	}
 	env.workflowDef = workflowDefinition
+	header := &shared.Header{
+		Fields: map[string][]byte{
+			"test-header": []byte("test-data"),
+		},
+	}
```

Comment:
What is the purpose of this header for workflow testing?

Reply:
Removed, made this configurable so user can use on tests..